client/src/components/pages/home/extra-work.tsx

```diff
@@ -1,13 +1,18 @@
+import ExtraWorkCard from "./extra-work-card";
 import "./extra-work.module.css";
+import { extraWorkData } from "@/data/extra-work-data";
 
 const ExtraWork = () => {
-  return (
-    <div className="extra-work">
-      <div className="w-full">
-        <h1>Extra Work</h1>
-      </div>
-    </div>
-  );
+	return (
+		<div className="extra-work p-[32px] overflow-x-scroll">
+			<h2 className="text-2xl font-bold">Check out more of our work!</h2>
+			<div className="flex w-full pt-[16px] gap-x-[64px]">
+				{extraWorkData.map(item => (
+					<ExtraWorkCard item={item}/>
+				))}
```

Comment:
We are currently getting the following console error:
```
hook.js:608 Each child in a list should have a unique "key" prop.

Check the render method of `ExtraWork`. See https://react.dev/link/warning-keys for more information.
```

Giving each card a unique id will resolve that issue:

```suggestion
				{extraWorkData.map((item, index) => (
					<ExtraWorkCard key={index} item={item}/>
				))}
```